src/js/constants/BlackListServiceConfig.js

```diff
@@ -0,0 +1,16 @@
+let BlackListServiceConfig = [
+  'uris',
+  'ports',
+  'version',
+  'versionInfo',
+  'deployments',
+  'tasks',
+  'taskStats',
+  'tasksHealthy',
+  'tasksRunning',
+  'tasksStaged',
+  'tasksUnhealthy',
+  '_itemData'
```

Comment:
It this necessary? Isn't this an effect of the bug you discovered here: https://github.com/dcos/dcos-ui/pull/903/files#diff-0eb72bc636980e431e5fe14b2a54616dR51